test/OpenFeature.Tests/OpenFeatureEventTests.cs

```diff
@@ -0,0 +1,413 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using NSubstitute;
+using OpenFeature.Constant;
+using OpenFeature.Model;
+using OpenFeature.Tests.Internal;
+using Xunit;
+
+namespace OpenFeature.Tests
+{
+    [SuppressMessage("Reliability", "CA2007:Consider calling ConfigureAwait on the awaited task")]
+    public class OpenFeatureEventTest : ClearOpenFeatureInstanceFixture
+    {
+        [Fact]
+        [Specification("5.1.2", "When a `provider` signals the occurrence of a particular `event`, the associated `client` and `API` event handlers MUST run.")]
+        public async Task Event_Executor_Should_Propagate_Events_ToGlobal_Handler()
+        {
+            var eventHandler = Substitute.For<EventHandlerDelegate>();
+
+            eventHandler.Invoke(Arg.Any<ProviderEventPayload>());
```

Comment:
I'm not sure I understand why you invoke this here. Are you just testing that it can be?

Reply:
Seems like that was a copy-paste leftover. I will remove this again